src/diffusers/loaders/ip_adapter.py

```diff
@@ -116,6 +119,22 @@ def load_ip_adapter(
         local_files_only = kwargs.pop("local_files_only", None)
         token = kwargs.pop("token", None)
         revision = kwargs.pop("revision", None)
+        low_cpu_mem_usage = kwargs.pop("low_cpu_mem_usage", _LOW_CPU_MEM_USAGE_DEFAULT)
+
+        if low_cpu_mem_usage and not is_accelerate_available():
```

Comment:
I think we only need to  throw these warnings once inside `_convert_ip_adapter_attn_to_diffusers` and `_convert_ip_adapter_image_proj_to_diffusers`;

Reply:
Or should we only throw it from `load_ip_adapter()` since the entry points to `_convert_ip_adapter_attn_to_diffusers()` and `_convert_ip_adapter_image_proj_to_diffusers()` are supposed to be from `load_ip_adapter()`. WDYT?

Reply:
sure:)